stable/phabricator/templates/deployment.yaml

```diff
@@ -19,13 +19,26 @@ spec:
         app: {{ template "phabricator.fullname" . }}
         chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
         release: "{{ .Release.Name }}"
+{{- if or .Values.podAnnotations .Values.metrics.enabled }}
```

Comment:
`podAnnotation` is not defined in `values.yaml` (and `values-production.yaml` if it exists).

Can you add it and also include the new variable in the README?